adoc/admin-ses-integration.adoc

```diff
@@ -87,36 +91,45 @@ spec:
       readOnly: false
 *EOF*
 ----
-. Verify that the pod exists and its status: 
+//TODO: Where do I get the input for POD_NAME, CONTAINER_NAME and IMAGE_NAME?
```

Comment:
POD_NAME is used as K8s pod resource identifier, could be anything you like.
CONTAINER_NAME is used as K8s container identifier could be any name you like.
IMAGE_NAME is the image to use for the container. This is the same as image used for docker,  for example opensuse/leap.

Reply:
> IMAGE_NAME is the image to use for the container. This is the same as image used for docker, for example opensuse/leap.

the same image_name as is used in the `terraform.tvars` file? https://susedoc.github.io/doc-caasp/master/caasp-deployment/single-html/#_deploying_the_cluster_nodes

Reply:
> > IMAGE_NAME is the image to use for the container. This is the same as image used for docker, for example opensuse/leap.
> 
> the same image_name as is used in the `terraform.tvars` file? https://susedoc.github.io/doc-caasp/master/caasp-deployment/single-html/#_deploying_the_cluster_nodes

Negative. By here user should have already completed the deployment of CaaSP cluster and SES cluster. The IMAGE_NAME would be to user's choice of container image. it could be any that the user wish to deploy the pod with. 

For example when the IMAGE_NAME is opensuse/leap (or could be any other image such as busybox, or alpine...etc.), the kubernetes will deploy a pod(container) from opensuse/leap container image. And within this pod it will have the RBD mounted.